src/FINDOLOGIC/Export/Helpers/Serializable.php

```diff
@@ -15,7 +15,9 @@
     public function getDomSubtree(\DOMDocument $document);
 
     /**
+     * @param array $availableProperties Properties that are available across the data set, so an individual item
+     *      knows into which column to write its property value, if any.
      * @return string A CSV fragment that, combined with other fragments, will finally become an export file.
      */
-    public function getCsvFragment();
+    public function getCsvFragment(array $availableProperties = []);
```

Comment:
`$availableProperties = []` is only used within the `CSVItem`-Class could be removed from all methods IMHO. If there is any other use. Let me know

Reply:
For a consistent class structure, I'd prefer to keep it.